pkg/api/serve_test.go

```diff
@@ -147,7 +148,15 @@ func setupClientByEndpoint(t testing.TB, endpointURL string, accessKeyID, secret
 
 func setupServer(t testing.TB, handler http.Handler) *httptest.Server {
 	t.Helper()
-	server := httptest.NewServer(http.TimeoutHandler(handler, ServerTimeout, `{"error": "timeout"}`))
+	withServerTimeout, err := strconv.ParseBool(os.Getenv("TEST_WITH_SERVER_TIMEOUT"))
+	if err != nil {
+		// by default we test with server timeout
+		withServerTimeout = true
+	}
```

Comment:
I would panic here instead of allowing `TEST_WITH_SERVER_TIMEOUT=NO` to be parsed as wanting server timeout. Or at least log the error. It's just a test so non-blocking.

Reply:
can panic on invalid value - I was thinking that the main use case is the default - empty